### ISSUE TITLE
Embedding Applozic Chat

### ISSUE THREAD

**Issue by bradmartin** [AppLozic/Applozic-NativeScript-Chat-Plugin#2]
So currently the plugin is launching the activity on android from the SDK which is great. I'm becoming familiar with Applozic and trying to determine the capability of embedding the applozic UI inside a custom UI in android/iOS, specifically with NativeScript.  I came across [this doc](https://docs.google.com/document/d/1qH1razXLCg_aCWgdhVp4ATCAZqU3APugWiUbSEJm0-A/edit) which describes adding chat fragments to an activity. 

Adding a fragment with NS is possible in context of a plugin also as seen here: https://github.com/TeamMaestro/nativescript-pspdfkit/blob/master/src/android/activity.ts#L4

I can help move this forward in the direction of allowing multiple options, to either launch the activity (leave the current design) but also extend with the option of creating a custom native view to embed the fragment inside an NS app where needed.

Hope this makes sense, if not feel free to continue the discussion 😄

**Comment by bradmartin** [comment]
I'm not very knowledgeable with iOS but skimming the docs it looks like https://www.applozic.com/docs/ios-chat-sdk.html#container-view this is the approach to embed the UI inside an existing iOS application. Is that correct?

**Comment by mariano-aguero** [comment]
Hi @bradmartin Do you want to integrate it into a custom UI because you want to modify its appearance? There is some customization in the docs https://docs.applozic.com/docs/mobile-chat-theme-and-customization . Maybe can help you

**Comment by bradmartin** [comment]
Not necessarily changing the theme/style but actually embedding it into another activity on android (and if there is an iOS equivalent). Right now, I believe the only way this works is it opens the Applozic chat activity in android apps. I reviewed the one documentation about using the fragment, which is something NS supports. I started to work on this on a fork I have by keeping the current work done as the main plugin class component. Then adding a class for a `view` component which would be the fragment that can be put into a layout container in the Android app. 

I'm short on time right now but let me know if that makes any sense :)
